src/components/Playlists.css

```
@@ -33,3 +33,141 @@
+  color: white;
+}
+
+.playlist-header {
+  margin-bottom: 30px;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.1);
+  padding-bottom: 20px;
+}
+
+.playlist-title {
+  font-size: 32px;
+  font-family: 'GothamBold', sans-serif;
```

Stylelint (17.2.0)

[error] Unexpected quotes around "GothamBold" (font-family-name-quotes)

(font-family-name-quotes)